Assets/PlanningSceneManager.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlanningSceneManager : MonoBehaviour
+{
+    public Canvas gameCanvas;
+
+    private static PlanningSceneManager _instance;
+    public static PlanningSceneManager Instance
+    {
+        get
+        {
+            if( _instance == null)
+            {
+                Debug.LogError("Planning Scene Manager is NULL!");
+            }
+            return _instance;
+        }
+    }
+
+    private void Awake()
+    {
+        _instance = this;
```

Comment:
I think this might happen automatically? I'm not really sure though

Reply:
sadly it does not :( this is how you can sort of fake Singletons in unity